mhkit/river/io/d3d.py

```diff
@@ -461,7 +542,7 @@ def turbulent_intensity(data, points='cells', time_index= -1,
     TI_data.loc[zero_ind,'turkin1']=np.zeros(len(zero_ind))
```

Comment:
L498: Additionally we should check that the provided time index is within the range of the dataset